### PR TITLE
Added support for generating artifacts of specific packs

### PR DESCRIPTION
**Pull request by guykeller**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/25442#issuecomment-647139021

## Description
Added logic that will allow specifying which content packs  should be added to the artifacts.

### CODE REVIEW

File: demisto_sdk/__main__.py [hunk 1]
```diff
@@ -303,6 +303,10 @@ def validate(config, **kwargs):
     '--packs', is_flag=True,
     help='If passed, will create only content_packs.zip'
 )
+@click.option(
+    '-pi', '--packs_to_include',
```
Comment (ronykoz) [inline demisto_sdk/__main__.py:307]:
```suggestion
    '-p', '--packs_to_include',
```

File: demisto_sdk/commands/create_artifacts/content_creator.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ class ContentCreator:
 
     def __init__(self, artifacts_path: str, content_version='', content_bundle_path='',
                  test_bundle_path='', packs_bundle_path='', preserve_bundles=False, packs=False,
-                 no_update_commonserver=False):
+                 no_update_commonserver=False, packs_to_include=[]):
```
Comment (ronykoz) [inline demisto_sdk/commands/create_artifacts/content_creator.py:43]:
```suggestion
                 no_update_commonserver=False, packs_to_include=''):
```
default list might cause problems

**Comment by ronykoz** [comment]
why did you rechange to pi again?

**Comment by guykeller** [comment]
@ronykoz -p is taken. Changing it to -i

File: demisto_sdk/__main__.py [hunk 3]
```diff
@@ -303,7 +303,12 @@ def validate(config, **kwargs):
     '--packs', is_flag=True,
     help='If passed, will create only content_packs.zip'
 )
+@click.option(
+    '-i', '--packs_to_include',
```
Comment (ronykoz) [inline demisto_sdk/__main__.py:307]: then remove -i option altogether

File: demisto_sdk/__main__.py [hunk 4]
```diff
@@ -303,7 +303,12 @@ def validate(config, **kwargs):
...
+    help='A comma separated list of specific packs to include. If passed, no other packs will be  added.'
+)
 def create(**kwargs):
+    print("TESTING")
```
Comment (ronykoz) [inline demisto_sdk/__main__.py:311]: delete

**Comment by orhovy** [comment]
@guykeller Not sure it is relevant anymore, as Gal Rabins pr on content artifacts - https://github.com/demisto/demisto-sdk/pull/625/files#diff-127993170afed108dd1f5ed8c45b50f6? 
WDUT? Can be closed?

**Comment by guykeller** [comment]
@orhovy Correct, closing.

**Pull request closed without merging**